From fd7a7942499636c5cf40575568e2dfff8d80dec2 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Thu, 26 Mar 2020 08:28:29 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10: fix SD UHS-I support

Limit max-frequency to 100MHz in UHS-I mode and fix
vccio_sd.

Change-Id: Ide7eb3591415093c363d1802d7ff19bbf22b6dd1
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index bf85b396d768..046c8444d206 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -358,7 +358,7 @@
 			vccio_sd: LDO_REG8 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
+				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <3300000>;
 				regulator-name = "vccio_sd";
 				regulator-state-mem {
@@ -504,11 +504,13 @@
 	cap-mmc-highspeed;
 	cap-sd-highspeed;
 	card-detect-delay = <200>;
+	max-frequency = <100000000>;
 	rockchip,default-sample-phase = <90>;
 	supports-sd;
 	sd-uhs-sdr12;
 	sd-uhs-sdr25;
 	sd-uhs-sdr104;
+	vqmmc-supply = <&vccio_sd>;
 	status = "okay";
 };
 
-- 
2.35.3

